### PR TITLE
Add tuple tests to portable47

### PR DESCRIPTION
**Pull request by KevinRansom**
Now that we have a tuple dll that supports portable47, enable the tuple tests in core unit tests for the profile47 version of fsharp.core.dll

### CODE REVIEW

File: src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj [hunk 1]
```diff
@@ -93,19 +94,22 @@
...
   <ItemGroup>
     <Compile Include="LibraryTestFx.fs" />
+    <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Utils.fs" Condition=" '$(TargetFramework)' != 'portable47' " />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Utils.fs" Condition="'$(TargetFramework)' != 'portable47'" />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\ArrayModule.fs" />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\ArrayModule2.fs" />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Array2Module.fs" />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Array3Module.fs" />
     <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Array4Module.fs" />
+    <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\ArrayProperties.fs" Condition=" '$(TargetFramework)' != 'portable47' " />
```
Comment (dsyme) [inline src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj]: Duplicate line?

File: src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj [hunk 2]
```diff
@@ -93,19 +94,22 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="LibraryTestFx.fs" />
+    <Compile Include="FSharp.Core\Microsoft.FSharp.Collections\Utils.fs" Condition=" '$(TargetFramework)' != 'portable47' " />
```
Comment (dsyme) [inline src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj]: Duplicate line?

**Comment by dsyme** [comment]
Looks good

File: src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj (hunk 2 above)
Comment (KevinRansom) [inline src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj]: yep

File: src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj [hunk 3]
```diff
@@ -59,12 +59,13 @@
       <Project>{DED3BBD7-53F4-428A-8C9F-27968E768605}</Project>
       <Name>FSharp.Core</Name>
     </ProjectReference>
-    <Reference Include="System.ValueTuple" Condition="'$(TargetFramework)' != 'portable47'">
+    <Reference Include="System.ValueTuple">
         <HintPath Condition=" '$(TargetFramework)' == 'coreclr' ">..\..\..\packages\System.ValueTuple.4.0.0-rc3-24212-01\lib\netstandard1.1\System.ValueTuple.dll</HintPath>
-        <HintPath Condition=" '$(TargetFramework)' != 'coreclr' ">..\..\..\packages\System.ValueTuple.4.4.0-beta-24631-01\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+        <HintPath Condition=" '$(TargetFramework)' == 'profile47' ">..\..\..\packages\System.ValueTuple.4.4.0-beta-24631-01\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
```
Comment (dsyme) [inline src/fsharp/FSharp.Core.Unittests/FSharp.Core.Unittests.fsproj]: Why all these different versions?  Why not just one version?

**Pull request merged**